src/main/java/de/hysky/skyblocker/skyblock/waypoint/WaypointsListWidget.java

```diff
@@ -143,7 +150,12 @@ public WaypointGroupEntry(WaypointGroup group) {
                 WaypointsListWidget.this.children().remove(this);
                 waypoints.remove(group);
             }).width(38).build();
-            children = List.of(enabled, nameField, ordered, buttonNewWaypoint, buttonDelete);
+			Text arrow = Text.of(collapsed ? "▼" :"▲");
+			collapseWaypoint = ButtonWidget.builder(arrow, button -> {
```

Comment:
This really should be first in both the method and the children list, but this is not a big deal.